packages/fetchai/skills/tac_control_contract/game.py

```diff
@@ -87,8 +87,9 @@ def __init__(
         self._version_id = version_id
         self._tx_fee = tx_fee
         self._agent_addr_to_name: Dict[str, str] = defaultdict()
+        # The next  line creates a problem when we are trying to generate ids due to the fact that we haven't initialise the contract.
         self._good_id_to_name = generate_good_id_to_name(nb_goods, contract)
-        #  self._check_consistency()
+        # self._check_consistency()
```

Comment:
Commented out the check consistency at the __init__ function. I am checking the consistency before we start the game.